chema-cabeza/Precurso/Bingo/bingo.js

```diff
@@ -0,0 +1,305 @@
+let bingoCard = [
+
+    //Linea 1
+    [ {number:  0, matched: false },
+         
+     { number:  0, matched: false },
+  
+     { number:  0, matched: false },
+  
+     { number:  0, matched: false },
+  
+     { number:  0, matched: false }],
+      
+     //Linea 2 
+     [{number:  0, matched: false },
+  
+     { number:  0, matched: false },
+      
+     { number:  0, matched: false },
+  
+     { number:  0, matched: false },
+  
+     { number:  0, matched: false }],
+     
+     //Linea 3 
+     [{ number: 0, matched: false },
+  
+     { number:  0, matched: false },
+         
+     { number:  0, matched: false },
+     
+     { number:  0, matched: false },
+     
+     { number:  0, matched: false }]
+ 
+
+];
+
+//I'm using global variables/arrays this time since many function used them repeatedly
+
+let playerRecords = [];  // To
+
+let usedCardNumbers =[];
+ 
+ let turns;
+ 
+ let line1Flag;
+ let line2Flag;
+ let line3Flag;
+
+ let playerName = "";
+ 
+
+(window.bingo = function() {
+
+    turns = 0;
+
+    usedNumbers = [];
+
+    line1Flag = false;
+    line2Flag = false;
+    line3Flag = false;
+
+    if (playerRecords.length > 0)  {printScore()} // Avoids printing the score table if it's empty
+
+    playerName = prompt (`Please, introduce your name:`);
+
+
+        if (playerName === null) {return false;}
+        else {console.log (`Hi ${playerName}, here's your card:`)}
+        
+    newCard();
+
+})();
+
+function printScore(){
+
```

Comment:
como anteriormente, intentar no dejar espacio tras declaraciones de if, else, function, for....